CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 
 ### Added
 
+- Implemented BackupManagerGit to handle automatic backups of .bib files using Git, ensuring centralized, version-controlled backup management. [#2961](https://github.com/JabRef/jabref/issues/2961)
+- Added automatic copying of .bib files to a jabref/backups directory every 19 seconds with comprehensive commit history. [#2961](https://github.com/JabRef/jabref/issues/2961)
+- Added support for unique file naming using UUIDs to prevent overwriting files with identical names in the backup directory. [#2961](https://github.com/JabRef/jabref/issues/2961)
```

Comment:
@khola22 Please do not mark review comments as resolved (except if it is trivially visible that it has been resolved). In this case, this was asked to be removed, and it is not (yet).
Once you push fresh commits on the code window, this will automatically be marked "Outdated". Then, one of the maintainers can take a look and mark them resolved.
This also allows for conversations in case you get stuck in a particular comment's context or if we decide on any change of plans.

Reply:
@subhramit okay ! sorry about that .

Reply:
> @subhramit okay ! sorry about that .

No issues, and no need to be sorry. It's on us to guide new contributors!